app/src/org/commcare/activities/CommCareHomeActivity.java

```diff
@@ -172,10 +172,6 @@ protected void onCreateSessionSafe(Bundle savedInstanceState) {
         processFromExternalLaunch(savedInstanceState);
         processFromShortcutLaunch();
         processFromLoginLaunch();
-
-        if (CommCareApplication._().isConsumerApp()) {
```

Comment:
Decided with Clayton that we're not going to do anything with the new restore file on upgrades